locales/ar_SA/pcsx2_Main.po

```diff
@@ -1175,33 +1170,31 @@ msgstr "إختر ملف لإعدادات %s"
 msgid "Settings"
 msgstr "الإعدادات"
 
-#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:86
+#: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:84
```

Comment:
The new line changes seem incorrect to me.

Reply:
Maybe he used an outdated file? Just the files i got from Plubu in Discord. I'll correct it.

Reply:
That line says pcsx2/gui/Dialogs/FirstTimeWizard.cpp:48. If anyone can look into this please.

Reply:
86 is correct